apps/academy/src/pages/tracks/nft-solidity/5.mdx

```diff
@@ -14,7 +14,7 @@ import LessonInformationalModal from "../../../components/mdx/LessonInformationa
 
 <LessonLayout
   lessonTitle="Connect your Smart Contract to a Front End Application."
-  author="piablo"
```

Comment:
I assume this should be `ropats16` to also match the Twitter?

Reply:
Yes, for sure. thanks :) We added new components, and they got populated with 'something' or 'someone'. They all needed to be reviewed.